feat(actions): Add open_in_mini_files action

wimmstr opened the pull request:
Provides an alternative to open_in_filebrowser()

I guess it does not handle setting mini.files as dependency.
I could add a line regarding this in the documentation if you want.


I hope this is ok, as this is my first PR. I just added an alternative function to open files with mini files. 

Thanks for this plugin!

kind regards

Commit:
feat(actions): Add open_in_mini_files action

Provides an alternative to open_in_filebrowser

tsakirist commented:
> Provides an alternative to open_in_filebrowser()
> 
> I guess it does not handle setting mini.files as dependency. I could add a line regarding this in the documentation if you want.
> 
> I hope this is ok, as this is my first PR. I just added an alternative function to open files with mini files.
> 
> Thanks for this plugin!
> 
> kind regards

Thanks for the PR.

Please make the necessary changes:

- Extend [`config.lua`](https://github.com/tsakirist/telescope-lazy.nvim/blob/main/lua/telescope/_extensions/lazy/config.lua) with the keymap, add it below `open_in_file_browser`
- Extend [`README.md`](https://github.com/tsakirist/telescope-lazy.nvim/blob/main/README.md) with the optional dependency

Commit:
fix(config, README): Add open_in_mini_files default keymap; Add keymap and optional dependency to README

wimmstr commented:
Added the necessary changes.

Decided on M-f for the Keybinding. Can change it to whatever you prefer.
I also added the keymapping in the two places of the readme where keymappings are shown (default config, and keymapping table).

kind regards!

Commit:
fix(README): Table-border spacing

tsakirist commented:
Sorry for the long delay, I was on vacation!

The PR looks good, but after considering this functionality, I've decided to maintain the focus of this plugin solely on **telescope**, as its name implies (otherwise, I'd have to change the plugin name as well). Incorporating **non-telescope** plugins for some actions falls outside its intended scope.

I’ll close this for **now**. Sorry for any inconvenience.

wimmstr commented:
No problem whatsoever!
